### PR TITLE
"Go to payment" button

### PR DESCRIPTION
**Pull request by kharann**
Added button which redirects you to the payment page to the following event if it has payment.

Also fucked a bit around with the styling for positioning of the blocks

### CODE REVIEW

**Comment by kharann** [comment]
Kinda fixes #399

**Review by aslakhol — APPROVED** [review]
LGTM

**Comment by aslakhol** [comment]
Fix tester også er gogo 🔥

File: src/events/components/Attendance/UnattendButton.tsx [hunk 1]
```diff
@@ -15,16 +15,16 @@ const UnattendButton: FC<IAttendButtonProps> = ({ eventId, isOnWaitList, waitLis
 
   if (!isOnWaitList) {
     return (
-      <div>
+      <>
         <Button onClick={signOff}>Meld meg av</Button>
-      </div>
+      </>
```
Comment (oleast) [inline src/events/components/Attendance/UnattendButton.tsx:20]: Don't need the fragment here anymore 😁

**Review by oleast — APPROVED** [review]
LGTM

**Pull request merged**